assets/styl/_base/form-elements.twig

```diff
@@ -0,0 +1,51 @@
+<form xmlns="http://www.w3.org/1999/html">
+    <div class="pb1">
```

Comment:
Usualy we use a container for label and input
```html
<div class="form-field">
    <label for="input-id">Label : </label>
    <input id="input-id" type="text"/>
</div>
```
a paragraphe should contain complete sentences.

W3C advice to use

```html
<p class="form-field">
    <label for="input-id">Label : </label>
    <input id="input-id" type="text"/>
</p>
```

But by experience a field contain more than only label and input (Error messages, help informations, …).
And this can be <p> and <ul> so forbidden in a <p>